api/v1alpha1/labels.go

```diff
@@ -16,4 +16,6 @@ const (
 	LabelTrueValue = "true"
 
 	FinalizerName = "kargo.akuity.io/finalizer"
+
+	AllowSharedOwnershipLabelKey = "kargo.akuity.io/allow-shared-ownership"
```

Comment:
This is quite lengthy and cumbersome to type. Given this, I am wondering if it could for example be changed to `kargo.akuity.io/shared-owner`. Which gets the same message across.